core/src/test/scala/integration/kafka/api/SaslSslAdminIntegrationTest.scala

```diff
@@ -519,6 +523,49 @@ class SaslSslAdminIntegrationTest extends BaseAdminIntegrationTest with SaslSetu
     }
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testExpireDelegationToken(quorum: String): Unit = {
+    client = createAdminClient
+    val createDelegationTokenOptions = new CreateDelegationTokenOptions()
+
+    // Test expiration for non-exists token
+    TestUtils.assertFutureExceptionTypeEquals(
+      client.expireDelegationToken("".getBytes()).expiryTimestamp(),
+      classOf[DelegationTokenNotFoundException]
+    )
+
+    // Test expiring the token immediately
+    val token1 = client.createDelegationToken(createDelegationTokenOptions).delegationToken().get()
+    val expireTokeOptions = new ExpireDelegationTokenOptions()
+    // wait for broker sync
+    Thread.sleep(500)
+    val token1ExpireTime = assertDoesNotThrow(() => client.expireDelegationToken(
+      token1.hmac(),
+      expireTokeOptions.expiryTimePeriodMs(-1)).expiryTimestamp().get()
+    )
+    assertTrue(token1ExpireTime < System.currentTimeMillis())
+
+    // Test expiring the expired token
+    val token2 = client.createDelegationToken(createDelegationTokenOptions).delegationToken().get()
+    // Ensure current time > maxLifeTimeMs of token
+    Thread.sleep(5000)
```

Comment:
This sleep time is too large. Could you please create a token with small `maxLifeTimeMs` instead?